dashboard/lib/test_section.rb

```diff
@@ -0,0 +1,239 @@
+include FactoryBot::Syntax::Methods
+
+class TestSection
+  DEFAULT_TEACHER_EMAIL = 'test_teacher@code.org'
+  DEFAULT_TEACHER_PASSWORD = 'test_password'
+  DEFAULT_TEACHER_NAME = 'Teacher Generated'
+  DEFAULT_SECTION_NAME = 'Generated Test Section'
+  DEFAULT_NUM_STUDENTS = 10
+
+  SAMPLE_STUDENT_NAME_FORMAT = 'Student%s Generated'.freeze
+  SAMPLE_STUDENT_NAME_REGEX = /Student\d* Generated/
+
+  DEFAULT_UNIT = 'csp3-2024'
```

Comment:
Did you have a specific reason to pick this unit? I don't think it ultimately matters too much, but the rest of the Evaluate work is currently in CSP unit 4.

Reply:
Just mis-remembering which unit we were working with